### PR TITLE
R4R: Add APIs in LCD

### PR DESCRIPTION
**Pull request by HaoyangLiu**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ services:
     - "26656-26657:26656-26657"
     volumes:
     - ./build:/home
-    command: /bin/bash -c 'cd /home/nodecluster/node0 && ../../iris start --home iris --log_level info'
+    command: /bin/bash -c 'cd /home/nodecluster/node0 && ../../iris start --home iris --log_level error'
```
Comment (HaoyangLiu) [inline docker-compose.yml]: Revert this change before R4R

**Comment by haifengxi** [comment]
https://github.com/irisnet/irishub/issues/938
